packages/base/adapters/evm/wagmi/index.ts

```diff
@@ -0,0 +1,13 @@
+import '@web3modal/polyfills'
+
+export { EVMWagmiClient } from './client.js'
+
+// -- Types
+export type { AdapterOptions } from './client.js'
+
+// -- Utils
+export { defaultWagmiConfig as defaultWagmiCoreConfig } from './utils/defaultWagmiCoreConfig.js'
```

Comment:
What about unifying them? Or we could do this in another PR.

Reply:
Just to keep the PR small, `186 - 1` :p

Reply:
I say let's keep just one